File: Fronted/gestion-tareas/src/app/modules/tareas/application/dtos/response/GestionTareasResponse.ts
import { EstadoModel } from "src/app/core/models/EstadoModel";
import { PrioridadModel } from "src/app/core/models/PrioridadModel";

export class GestionTareasResponse {
    constructor(
      public idTarea?: number,
      public tituloTarea?: string,
      public descripcionTarea?: string,
      public fechaVencimiento?: Date,
      public estado?: EstadoModel,
      public prioridad?: PrioridadModel,
      public createAt?: Date,
      public updateAt?: Date
    ) {}
  }

File: Backend/gestion-tareas/gestion-tareas/DTOs/Response/GestionTareasResponse.cs
using gestion_tareas.Models;

namespace gestion_tareas.DTOs.Response
{
    public class GestionTareasResponse
    {
        public long? idTarea { get; set; }
        public string? tituloTarea { get; set; }
        public string? descripcionTarea { get; set; }
        public DateTime? fechaVencimiento { get; set; }
        public Estado? estado { get; set; }

        public string? estadoTarea { get; set; }
        public Prioridad? prioridad { get; set; }
        public DateTime? createAt { get; set; }
        public DateTime? updateAt { get; set; }
    }
}
